commitlint.config.ts

```diff
@@ -558,7 +560,27 @@ module.exports = {
                         `Please use full URLs instead of #XYZ refs.`
                     ];
                 },
+    
+                'proper-revert-message': ({header, body}: {header: any, body: any}) => {
+                    let offence = false;
+
+                    // does msg have a body?
```

Comment:
@realmarv I think this comment is wrong (it seems the next line after is not doing what the comment says)

Reply:
Done